enterprise/install-scripts/openshift.ks

```diff
@@ -1322,7 +1332,9 @@ configure_subscription()
    need_infra_repo && roles="$roles --role broker"
    need_client_tools_repo && roles="$roles --role client"
    need_node_repo && roles="$roles --role node"
-   need_jbosseap_cartridge_repo && roles="$roles --role node-eap"
+   need_jbosseap_cartridge_repo && { [[ -n "${jbosseap_version}" ]] && \
+                                         roles="$roles --role node-eap-${jbosseap_version}" || \
```

Comment:
If you keep the single `$jbosseap_version`, it would be simpler here to use `need_jbosseap_cartridge_repo && roles="$roles --role node-eap${jbosseap_version:+-${jbosseap_version}}"`.